src/Compilers/Core/Desktop/CommandLine/CommonCommandLineArguments.cs

```diff
@@ -335,8 +335,16 @@ public IEnumerable<AnalyzerReference> ResolveAnalyzerReferences(IAnalyzerAssembl
 
         internal ImmutableArray<DiagnosticAnalyzer> ResolveAnalyzersFromArguments(string language, List<DiagnosticInfo> diagnostics, CommonMessageProvider messageProvider, TouchedFileLogger touchedFiles, IAnalyzerAssemblyLoader analyzerLoader)
```

Comment:
@shyamnamboodiripad @tmeschter I think the command line build perf will be fine for most scenarios as we are just eagerly loading the analyzer assemblies, they will eventually be loaded (pretty soon) when we create the analyzer driver.